qa/tasks/cephfs/filesystem.py

```diff
@@ -336,41 +336,13 @@ def newfs(self, name='cephfs', create=True):
     def status(self):
         return FSStatus(self.mon_manager)
 
-    def delete_all_filesystems(self):
+    @staticmethod
+    def delete_all_filesystems(mds_cluster):
```

Comment:
Hey,

I just saw this line will cause an issue in the Dashboard API tests:

```
2020-07-30 08:44:15,814.814 INFO:__main__:----------------------------------------------------------------------
2020-07-30 08:44:15,814.814 INFO:__main__:Traceback (most recent call last):
2020-07-30 08:44:15,814.814 INFO:__main__:  File "/home/jenkins-build/build/workspace/ceph-dashboard-pr-backend/qa/tasks/mgr/dashboard/helper.py", line 160, in setUpClass
2020-07-30 08:44:15,814.814 INFO:__main__:    cls.mds_cluster.delete_all_filesystems()
2020-07-30 08:44:15,814.814 INFO:__main__:TypeError: delete_all_filesystems() missing 1 required positional argument: 'mds_cluster'
2020-07-30 08:44:15,814.814 INFO:__main__:
2020-07-30 08:44:15,814.814 INFO:__main__:----------------------------------------------------------------------
2020-07-30 08:44:15,814.814 INFO:__main__:Ran 14 tests in 234.540s
2020-07-30 08:44:15,814.814 INFO:__main__:
2020-07-30 08:44:15,814.814 INFO:__main__:
```

https://jenkins.ceph.com/view/mgr-dashboard/job/ceph-dashboard-pr-backend/4690/

Reply:
I'll take a look, thanks for pointing out to it.

Reply:
Thank you! Let me know if I can help :)

Reply:
Fixed.

Reply:
Related review thread - https://github.com/ceph/ceph/pull/32581#discussion_r463008332